waspc/src/Wasp/AppSpec.hs

```diff
@@ -44,7 +44,8 @@ data AppSpec = AppSpec
     migrationsDir :: Maybe (Path' Abs (Dir DbMigrationsDir)),
     -- | Absolute path to the .env file in wasp project source. It contains env variables to be
     -- provided to the server only during the development.
-    dotEnvFile :: Maybe (Path' Abs File'),
+    dotEnvServerFile :: Maybe (Path' Abs File'),
+    dotEnvClientFile :: Maybe (Path' Abs File'),
```

Comment:
Proper comments have to be added for each field.

Reply:
Sounds good! Additionally, I wonder if it would flow slightly better having them named `serverDotEnvFile` and `clientDotEnvFile`? 🤔 Maybe we can let @Martinsos chime in there, he is the resident naming guru. :)

Reply:
Hah thanks for the title @shayneczyzewski :D! I like both, as you said `serverDotEnvFile` flows a bit nicer, but on the other hand `dotEnvSomethingFile` is closer to name file has on the disk + both dot env files then share same prefix. So if I had to choose I would be slightly in favor of `dotEnvServerFile` even, but very close, so both is fine with me really.